test/food-test.js

```diff
@@ -23,210 +25,220 @@ describe('Food Endpoints', function(){
+    it('updates the foods name and calories for a food', function(done) {
+      const id = 1
+      const food = { name: 'banana', calories: 10 }
+      const options = {
+        method: 'PUT',
+        body: food,
+        json: true,
+        url: `/api/v1/foods/${id}`
+      }
+      this.request(options, function(error, response, body) {
+        if (error) { done(error) }
 
-          assert.equal(parsedFood[0].id, pizzaId)
-          assert.equal(parsedFood[0].name, pizzaName)
-          assert.equal(parsedFood[0].calories, pizzaCalories)
-          assert.ok(parsedFood[0].created_at, pizzaCreatedAt)
-          assert.equal(parsedFood[1].id, bananaId)
-          assert.equal(parsedFood[1].name, bananaName)
-          assert.equal(parsedFood[1].calories, bananaCalories)
-          assert.ok(parsedFood[1].created_at, bananaCreatedAt)
+        Food.find(1)
+        .then( function(data) {
+          return data.rows[0]
+            }).then( function(food) {
+
+              assert.equal(food.name, 'banana')
+              assert.equal(food.calories, 10)
+            })
           done()
-        })
       })
     })
+    it('updates the foods name', function(done) {
+      const id = 1
+      const food = { name: 'banana' }
+      const options = {
+        method: 'PUT',
+        body: food,
+        json: true,
+        url: `/api/v1/foods/${id}`
+      }
+      this.request(options, function(error, response, body) {
+        if (error) { done(error) }
 
-    describe('POST /api/v1/foods', function() {
-      it('POSTs /api/v1/foods', function(done) {
-        const food = { name: 'apple', calories: 20}
-        const options = {
-          method: 'POST',
-          body: food,
-          json: true,
-          url: '/api/v1/foods'
-        }
-
-        const ourRequest = this.request(options, function(error, response, body) {
-          if (error) { done(error) }
-
-          assert.equal(body.name, 'apple')
-          assert.equal(body.calories, 20)
+        Food.find(1)
+        .then( function(data) {
+          return data.rows[0]
+            }).then( function(food) {
+              assert.equal(food.name, 'banana')
+            })
           done()
-        })
       })
     })
 
-    describe('PUT /api/v1/foods/:id', function() {
-      this.timeout(1000000)
-
-      beforeEach( function(done) {
-        Food.create('pizza', 100)
-          .then( function(){ done() })
-        })
+    it('updates the calories count', function(done) {
+      const id = 1
+      const food = { calories: 15 }
+      const options = {
+        method: 'PUT',
+        body: food,
+        json: true,
+        url: `/api/v1/foods/${id}`
+      }
+      const ourRequest = this.request(options, function(error, response, body) {
+        if (error) { done(error) }
 
-      afterEach( function(done) {
-        Food.destroyAll()
-        .then( function() { done() })
+        Food.find(1)
+        .then( function(data) {
+          return data.rows[0]
```

Comment:
For these intermediary type promises with one line returns, it can be helpful to switch to one-line syntax without `{}`.

E.g., 

```js
Food.find(1).then(data => data.rows[0])
  .then(food => {
    assert...
  })
```